enterprise/server/workflow/config/config.go

```diff
@@ -167,16 +168,33 @@ func NewConfig(r io.Reader) (*BuildBuddyConfig, error) {
 	return cfg, nil
 }
 
+const kytheDownloadURL = "https://storage.googleapis.com/buildbuddy-tools/archives/kythe-v0.0.67.tar.gz"
+
+func checkoutKythe() string {
```

Comment:
nit: I find both bash blocks more readable when it's a single block with `` because the spacing is more obvious

Reply:
PTAL